201-vnet-to-vnet-bgp/azuredeploy.json

```diff
@@ -0,0 +1,351 @@
+            }
+        },
+        "subnet1Prefix2": {
+            "type": "string",
+            "metadata": {
+                "description": "The prefix for the first subnet in the second VNET."
+            }
+        },
+        "gatewaySubnetPrefix2": {
+            "type": "string",
+            "metadata": {
+                "description": "The prefix for the GatewaySubnet where the second VirtualNetworkGateway will be deployed. This must be at least /29."
+            }
+        },
+        "gatewayPublicIPName2": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the PublicIP attached to the second VirtualNetworkGateway."
+            }
+        },
+        "gatewayName2": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the second VirtualNetworkGateway."
+            }
+        },
+        "connectionName2": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the second connection, connecting the second VirtualNetworkGateway to the first VirtualNetworkGateway."
+            }
+        },
+        "sharedKey": {
+            "type": "string",
+            "metadata": {
+                "description": "The shared key used to establish connection between the two VirtualNetworkGateways."
+            }
+        },
+        "asn1": {
+            "type": "int",
+            "metadata": {
+                "description": "The Autonomous System Number for the first VirtualNetworkGateway. Can be Private."
+            }
+        },
+        "asn2": {
+            "type": "int",
+            "metadata": {
+                "description": "The Autonomous System Number for the second VirtualNetworkGateway.Can be Private."
+            }
+        },
+        "gatewaySku": {
+            "type": "string",
+            "metadata": {
+                "description": "The size of the gateway. Note: only Standard & High Performance compatible at this time.",
+                "defaultValue": "Standard",
+                "allowedValues": [
+                    "Standard",
+                    "HighPerformance"
+                ]
+            }
+        }
+    },
+    "variables": {
+        "apiVersion": "2016-03-30",
+        "vnetID1": "[resourceId('Microsoft.Network/virtualNetworks',parameters('virtualNetworkName1'))]",
+        "gatewaySubnetRef1": "[concat(variables('vnetID1'),'/subnets/','GatewaySubnet')]",
+        "vnetID2": "[resourceId('Microsoft.Network/virtualNetworks',parameters('virtualNetworkName2'))]",
+        "gatewaySubnetRef2": "[concat(variables('vnetID2'),'/subnets/','GatewaySubnet')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "[variables('apiVersion')]",
```

Comment:
use a literal apiVersion, not a variable - vars break intellisense in JSON editors